backend/clap/src/main/java/zpi/ppea/clap/mappers/DtoMapper.java

```diff
@@ -47,6 +46,19 @@ public interface DtoMapper {
 
     PartialRatingRequest dtoToPartialRatingRequest(UpdatePartialRatingDto dto);
 
+    @Mapping(target = "questions", source = "questionsList")
+    StudyVisitDto toStudyVisitDto(StudyVisitResponse studyVisitResponse);
+
+    @Mapping(target = "content", source = "content")
+    @Mapping(target = "answers", source = "answersList")
+    QuestionDto toQuestionDto(Question question);
+
+    @Mapping(target = "description", source = "description")
```

Comment:
Same as above + it got renamed in the other PR. If it's removed, we don't need to care about the rename (at least in this place).